src/utils/styleUtils.js

```diff
@@ -50,7 +50,7 @@ export const pointStyle = {
 export const circleStyle = new Circle({
     ...pointStyle,
     fill: new Fill({
-        color: hexToRgba(white),
+        color: hexToRgba(red),
```

Comment:
This broke the style of the drawing feature selection which is white and should stay white. I suggest to create new redCircleStyle and rename this to whiteCircleStyle